example/20-componentconfig-gardenlet.yaml

```diff
@@ -35,16 +35,18 @@ controllers:
     concurrentSyncs: 20
     syncPeriod: 1h
     retryDuration: 12h
-#    `respectSyncPeriodOverwrite` specifies whether Shoot owners can
-#    mark their Shoots ignored (no reconciliation) or change their sync period.
-#    respectSyncPeriodOverwrite: true
-#    `reconcileInMaintenanceOnly` specifies whether Shoot reconciliations
-#    can only happen during their maintenance time window or not.
-#    reconcileInMaintenanceOnly: true
+  # `respectSyncPeriodOverwrite` specifies whether Shoot owners can
+  # mark their Shoots ignored (no reconciliation) or change their sync period.
+#   respectSyncPeriodOverwrite: true
+  # `reconcileInMaintenanceOnly` specifies whether Shoot reconciliations
+  # can only happen during their maintenance time window or not.
+#   reconcileInMaintenanceOnly: true
+  # `progressReportFrequency` specifies how frequent the progress of a shoot operation shall be reported in its status.
+#   progressReportFrequency: 5s
```

Comment:
Shouldn't we make this value configurable in the gardenlet chart? Ref `charts/gardener/gardenlet/charts/runtime/template/configmap-componentconfig.yaml`.

Reply:
Nice catch, thank you! Done now.